bundle/tests/scorecard/minikube-kuttl/ingress-certificate/02-errors.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Ingress
+apiVersion: extensions/v1beta1
+metadata:
+  name: ingress-with-cert-rc
```

Comment:
since this Ingress resource shouldn't exist at all, it'll be good to just check for the name and exclude the spec field